### PR TITLE
Senior exec

### PR DESCRIPTION
**Pull request by AhmedJamal93**

### CODE REVIEW

File: src/Components/Registration/FinalPage.js [hunk 1]
```diff
@@ -296,14 +342,34 @@ class FinalPage extends Component{
...
+
     handleSubmit = (event) => {
+        // if (this.state.senior_executive === true 
+        //     && (this.state.quarterly_meeting === ''
+        //     || this.state.meetings_frequency === '')){
+        //     this.setState({
+        //         dialogueOpen: true
+        //     });
+        //     return;
+        // }
```
Comment (ahmedhamodi) [inline src/Components/Registration/FinalPage.js:354]: meant to be commented?

**Comment by arsidada** [comment]
1. You have unused vars which is failing the CI tests. Please remove them:
```
[1] ./src/Components/Registration/FinalPage.js
[1]   Line 18:8:  'Membership' is defined but never used  no-unused-vars
[1]   Line 24:8:  'SE' is defined but never used          no-unused-vars
[1]   Line 25:8:  'MenuItem' is defined but never used    no-unused-vars
[1]
[1] ./src/Components/SeniorExec/SeniorExec.js
[1]   Line 2:8:  'CssBaseline' is defined but never used  no-unused-vars
```

**Comment by arsidada** [comment]
2. If you go to `http://localhost:3000/register/4` then hit the Senior Executive checkbox the very first time on the page, nothing happens. On subsequent clicks it starts showing the date boxes. But the first time it doesn't. Try it out

File: src/Components/Registration/FinalPage.js [hunk 2]
```diff
@@ -325,6 +352,131 @@ class FinalPage extends Component {
...
+  //           openStripe: true,
+  //         });
+  //       } else {
+  //         this.signUp(0, "FREE");
+  //         this.setState({
+  //           verified: true,
+  //         });
+  //       }
+  //     }
+  //   };
```
Comment (ahmedhamodi) [inline src/Components/Registration/FinalPage.js:424]: Should this be commented out?

Comment (AhmedJamal93) [reply src/Components/Registration/FinalPage.js:424]: sorry it shouldnt be there, i commented it out to check the code was working and forgot to take it out after, will take care of it, thanks

**Pull request closed without merging**